rare/components/tabs/games/env_vars.py

```diff
@@ -0,0 +1,98 @@
+from PyQt5.QtWidgets import QGroupBox, QTableWidgetItem
+from rare.ui.components.tabs.games.env_vars import Ui_env_vars_groupbox
+from rare.utils import config_helper
+from rare.shared import LegendaryCoreSingleton
+
+class EnvVars(QGroupBox, Ui_env_vars_groupbox):
+    def __init__(self, parent):
+        super(EnvVars, self).__init__(parent=parent)
+        self.setupUi(self)
+        self.env_vars_table.cellChanged.connect(self.create_new_row)
+        self.env_vars_table.cellChanged.connect(self.update_env_vars)
+        self.env_vars_table.verticalHeader().sectionClicked.connect(self.remove_row)
+        self.app_name = None
+        self.list_of_labels = []
+        self.core = LegendaryCoreSingleton()
+    
+    def import_env_vars(self):
+        self.env_vars_table.clearContents()
+
+        list_of_keys = []
+        list_of_values = []
+ 
+        # First, we try to get all keys and values from the config
+        try:
+            for key in self.core.lgd.config[f"{self.app_name}.env"]:
+                list_of_keys.append(key)
+            for i in list_of_keys:
+                list_of_values.append(self.core.lgd.config[f"{self.app_name}.env"][i])
+        except KeyError:
+            pass
+
+        # We count how many keys we have and insert new lines (as many as we need).
+        count = len(list_of_keys)
+        i = 0
+        for _ in range(count):
+            self.list_of_labels.append("-")
+            self.env_vars_table.insertRow(i)
+            i += 1
+
+        # We set the row count to the length of the list_of_keys
+        if len(list_of_keys) == 0:
+            self.env_vars_table.setRowCount(1)
+        else:
+            self.env_vars_table.setRowCount(len(list_of_keys))
+
+
+        # We always need to append one more minus at the end, because we want to have the new empty line to also have the minus.
+        self.list_of_labels.append("-")
+        self.env_vars_table.setVerticalHeaderLabels(self.list_of_labels)
+        for index, val in enumerate(list_of_keys):
+            new_item = QTableWidgetItem()
+            new_item.setText(val)
+            self.env_vars_table.setItem(index, 0, new_item)
+        
+        for index, val in enumerate(list_of_values):
+            new_item = QTableWidgetItem()
+            new_item.setText(val)
+            self.env_vars_table.setItem(index, 1, new_item)
+
+    def update_env_vars(self, row, column):
+        first_item = self.env_vars_table.item(row, 0)
+        second_item = self.env_vars_table.item(row, 1)
+        if column == 0 and first_item is not None and second_item is not None:
+            config_helper.add_option(f"{self.app_name}.env", first_item.text(), second_item.text())
+            config_helper.save_config()
+        if column == 1 and first_item is not None and second_item is not None:
+            config_helper.add_option(f"{self.app_name}.env", first_item.text(), second_item.text())
+            config_helper.save_config()
```

Comment:
What is the point of this? What else can column be instead of 0 or 1? Also what column does each index refer to?